test/unit/features/global-api/mixin.spec.ts

```diff
@@ -19,8 +19,8 @@ describe('Global API: mixin', () => {
   })
 
   it('should work for constructors created before mixin is applied', () => {
-    const calls = []
-    const Test = Vue.extend({
+    const calls: any[] = []
+    const Test: Vue = Vue.extend({
```

Comment:
do you need `: Vue` I would expect `Vue.extend ` to return `Vue`.

If this is necessary, don't need to change

Reply:
Yeah, typescript wasn't happy for every instance of this, but I didn't want to dig into the typing files just yet. I'll leave it for now, but putting a mental TODO